docs/API.md

```diff
@@ -50,4 +54,18 @@ For example: Accessing the data for a chat report will return the data if the re
 - Verbs should be unique and indicate the user's action (as perceived by the user). eg. `Request`, `Open`, `Accept`, `Pay`
   - If a unique verb cannot be used, then use only the standard verbs: `Update`, `Add`, `Delete`
 - Names must NOT include names of parameters eg. Bad: `SignInWithPasswordOr2FA` Good: `SignIn`
+- Names must NOT include the word Page or View, eg. Bad: `OpenReportPage` Good: `OpenReport`
 - Names must NOT reveal backend implementation details that the user does not know about eg. `AddVBBA` (users don't know what a VBBA is - verified business bank account) Good: `AddBankAccount`
+- For deep links like `StartAppWhileSignedInAndOpenWorkspace`, create separate commands for each specific scenario like `StartAppWhileSignedIn` and `OpenWorkspace`, where both requests will be triggered in parallel.
+
+## FAQ
+
+### How should error messages be persisted to Onyx?
+
+- The API layer Pusher onyxData should set specific error messages.
+- In cases where the API does not set an error, a generic error message can be set in the API.write onyxData `failureData` block.
+- Previous errors should be cleared in the `optimisticData` when a new request is made
```

Comment:
Maybe it would be a bit more consistent to say in API.write's `onyxData.failureData` and `onyxData.optimisticData`?

Since I'm not very familiarized I had to look in the docs that `optimisticData` was part of a parameter called `onyxData` we pass to `API.write`

Is there going to be a section at some point describing `API.write` and the parameter  like `onyxData`? or is that part of some other `.md` file?